tests/reals/CMakeLists.txt

```diff
@@ -0,0 +1,29 @@
+#Copyright (c) Microsoft. All rights reserved.
+#Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+cmake_minimum_required(VERSION 2.8.11)
+
+set(clds_reals_c_files
+    real_clds_hazard_pointers.c
+    real_clds_singly_linked_list.c
+    real_clds_sorted_list.c
+    real_clds_st_hash_set.c
+    real_lock_free_set.c
+)
+
+set(clds_reals_h_files
+    real_clds_hazard_pointers.h
+    real_clds_hazard_pointers_renames.h
+    real_clds_singly_linked_list.h
+    real_clds_singly_linked_list_renames.h
+    real_clds_sorted_list.h
+    real_clds_sorted_list_renames.h
+    real_clds_st_hash_set.h
+    real_clds_st_hash_set_renames.h
+    real_lock_free_set.h
+    real_lock_free_set_renames.h
+)
+
+include_directories(${CMAKE_CURRENT_LIST_DIR}/../../src)
+add_library(clds_reals ${clds_reals_c_files} ${clds_reals_h_files})
```

Comment:
>clds_reals [](start = 12, length = 10)

aren't the other ones called `reals_module_name` ? at least that's what i used for bs2sf : `reals_bs2sf`

Reply:
I'll rename reals_bs2sf to bs2sf_reals when I pull them into the main repo. I don't quite care about the name, but it's in c util like this and I feel its more to rename the one in c util.